config/routes.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Plz also check the code in the whole file to ensure that we won't break anything here 😄

Reply:
Oh, how would this break anything?

Reply:
this makes all string (well, not really, only literal ones) frozen, so if we make somewhere in this file something like `variable = 'hello'; variable << ' world'`, even if it is in a more subtle way, it will raise `FrozenError`now.

Reply:
It looks safe, I don't see any string variable declarations or modifications.